framework/CHANGELOG.md

```diff
@@ -38,10 +38,11 @@ Yii Framework 2 Change Log
 - Enh #13144: Refactored `yii\db\Query::queryScalar()` (Alex-Code)
 - Bug #13379: Fixed `applyFilter` function in `yii.gridView.js` to work correctly when params in `filterUrl` are indexed (SilverFire, arogachev)
 - Enh #13650: Improved `yii\base\Security::hkdf()` to take advantage of native `hash_hkdf()` implementation in PHP >= 7.1.2  (charlesportwoodii)
-- Bug #13379: Fixed `applyFilter` function in `yii.gridView.js` to work correctly when params in `filterUrl` are indexed (SilverFire) 
+- Bug #13379: Fixed `applyFilter` function in `yii.gridView.js` to work correctly when params in `filterUrl` are indexed (SilverFire)
```

Comment:
Duplicate line (see 2 lines above)  It's exactly the same except my name is added.